python/tests/test_framework/test_configs/slate.json

```diff
@@ -0,0 +1,130 @@
+[
+    {
+        "test_name": "slates",
+        "data_func": {
+            "name": "generate_slate_data",
+            "params": {
+                "num_examples": 1000,
+                "num_action": [
+                    3,
+                    3,
```

Comment:
[nit] maybe starting with 2x2 problem will make it more readable without actual coverage loss